src/libs/language-javascript-latest/describeApi.js

```diff
@@ -2,8 +2,11 @@ import { inspect } from 'util';
 import { stripIndent } from 'common-tags';
 import createError from '../error';
 
+import { registerBabel } from './_babelHelpers';
+
 export default function ({ rootDir }) {
   try {
+    registerBabel();
```

Comment:
we now require 'babel-register' only at this point, which should speed up the dawson CLI startup times and avoid unnecessary work.
Also, It would not make sense to use babel-register when using the python runtime (that will ship soon)